### PR TITLE
Build The Dashboard Page

### PR DESCRIPTION
**Pull request by Superfly101**
closes [Issue](https://github.com/Explore-Beyond-Innovations/ZeroCheck_Frontend/issues/7)

## Description
- Adds the Poppins font
- Creates reuseable card component
- Creates a dashboard components folder `app/components/dashboard/` for the dashboard
- Imports the Sidebar
- Adds the Active Events component
- Adds the Recent Activities component
- Adds the Upcoming Event component

### CODE REVIEW

File: app/components/sideNavOne.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const bottomNav = [
 ];
 const SideNavigation = () => {
   return (
-    <div className="hidden lg:block col-span-3 h-full bg-white overflow-y-scroll relative">
+    <div className="hidden lg:block min-w-[240px] min-h-screen bg-white overflow-y-scroll relative">
       <div className="pt-[58px] pl-[38px] pr-[15px]">
         <h4 className="text-3xl xl:text-4xl font-semibold text-black">
```
Comment (Ugo-X) [inline app/components/sideNavOne.tsx:60]: Not an exact replica of the design file
Screenshot 2024-12-20 at 10 51 44

File: app/components/dashboard/UpcomingEvents.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      imageUrl: "/images/event-4.png",
+    },
+  ];
+  return (
+    <Card>
+      <div className="px-2">
+        <div className="py-4 px-6">
+          <h2 className="font-semibold text-xl text-black">Upcoming Events</h2>
+        </div>
+        <div className="flex justify-between flex-wrap">
```
Comment (Ugo-X) [inline app/components/dashboard/UpcomingEvents.tsx:53]: missing a gray background as in the design file

File: app/components/sideNavOne.tsx (hunk 1 above)
Comment (Ugo-X) [reply app/components/sideNavOne.tsx:60]: also the side nav doesnt hightlight the active link as show in the design file

**Review by Ugo-X — CHANGES_REQUESTED** [review]

File: app/components/sideNavOne.tsx (hunk 1 above)
Comment (Superfly101) [reply app/components/sideNavOne.tsx:60]: I was quite hesitant to edit the Side nav and the event card components as I thought I only had to reuse them and not change them. I'll make the necessary changes now.

Comment (Superfly101) [reply app/components/sideNavOne.tsx]: The side nav now highlights the active link as show in the design file

**Review by Ugo-X — APPROVED** [review]

**Pull request merged**